src/components/common/MenuList.jsx

```diff
@@ -0,0 +1,50 @@
+import bag from 'assets/icon_bag.svg';
+import setting from 'assets/icon_setting.svg';
+import surfing from 'assets/icon_surfing.svg';
+import { React } from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+function MenuList() {
+  return (
+    <MenuContainer>
+      <MenuBtn>
+        <MenuIcon src={surfing} alt="서핏프로필 아이콘" />
+        서핏 프로필
+      </MenuBtn>
+      <MenuBtn>
+        <MenuIcon src={bag} alt="커리어관리 아이콘" />
+        커리어 관리
+      </MenuBtn>
+      <MenuBtn>
+        <MenuIcon src={setting} alt="설정 아이콘" />
+        설정
+      </MenuBtn>
+    </MenuContainer>
+  );
+}
+export default MenuList;
+
+const MenuContainer = styled.div`
+  width: 34.3rem;
+  margin: 1.6rem;
+  display: flex;
+  flex-direction: column;
```

Comment:
```suggestion
  width: 100%;
  display: flex;
  flex-direction: column;
  align-items: center;
```
이렇게 하면 MenuContainer가 전체 width 크기가 되는거고, 그리고 그 안에서 MenuBtn이 34.3rem 크기를 가지고 있으니까 중앙 정렬 시켜주면 자동으로 양 옆에 마진이 들어가게 돼! 이렇게 하면 이 메뉴 리스트를 재사용 했을때 드롭다운 박스 안에 메뉴리스트가 들어간다고 해도 width가 드롭다운 박스 크기에 맞춰서 100% 가 되고, 양옆에 마진도 자동으로 들어가서 재사용 할때 그대로 가져다가 쓸 수 있어! 그리고 화면 사이즈가 바뀌어도 영향을 받지 않는다는 장점도 있엉!